app/audit/models/ChangedLandlineNumberAuditModel.scala

```diff
@@ -19,30 +19,26 @@ package audit.models
 import play.api.libs.json.{JsValue, Json, Writes}
 import utils.JsonObjectSugar
 
-case class ChangedPhoneNumbersAuditModel(currentLandlineNumber: Option[String],
-                                         currentMobileNumber: Option[String],
-                                         requestedLandlineNumber: Option[String],
-                                         requestedMobileNumber: Option[String],
-                                         vrn: String,
-                                         isAgent: Boolean,
-                                         arn: Option[String]) extends ExtendedAuditModel {
+case class ChangedLandlineNumberAuditModel(currentLandlineNumber: Option[String],
+                                           requestedLandlineNumber: Option[String],
+                                           vrn: String,
+                                           isAgent: Boolean,
+                                           arn: Option[String]) extends ExtendedAuditModel {
 
   override val auditType: String = "ChangePhoneNumbers"
```

Comment:
Can you update the `auditType` to be `ChangeLandlineNumber` as this is what has been agreed on the TxM Confluence page